knowledge_repo/app/app.py

```diff
@@ -285,8 +286,15 @@ def format_date(date):
     def append_repo(self,name,uri):
```

Comment:
@abhirathb Where is this function being used? Can't see anywhere in the code where it is.